app/models/group.rb

```diff
@@ -84,11 +93,19 @@ def locked_members
   #
   # @return [Boolean] true if the group can be locked
   def lockable?
-    (members - locked_members).empty? && memberships_count == size
+    (members - locked_members).empty? && members_count == size
   end
 
   private
 
+  # override default attribute getter to include transfers
+  def members_count
+    return 0 unless memberships_count || transfers
+    return memberships_count unless transfers
+    return transfers unless memberships_count
```

Comment:
It was necessary to test shoulda_matchers, although I agree with you that there needs to be at least a leader so we should never have a memberships_count that's nil. I'll update.